src/utils/string_utils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @internal
+ * Removes leading zeros and explicit plus from textual representation of a number.
+ */
+export function removeLeadingZerosandExplicitPlus(str: string): string {
+  return !/[^+?0]+/.test(str)
```

Comment:
```suggestion
export function removeLeadingZerosAndExplicitPlus(str: string): string {
```
Or something more succinct `trimInsignificant`?